cmd/kubeadm/app/apis/kubeadm/v1beta4/types.go

```diff
@@ -749,6 +749,11 @@ type UpgradePlanConfiguration struct {
 	// +optional
 	DryRun *bool `json:"dryRun,omitempty"`
 
+	// EtcdUpgrade instructs kubeadm to execute etcd upgrade during upgrades.
+	// Defaults to true.
+	// +optional
+	EtcdUpgrade *bool `json:"etcdUpgrade,omitempty"`
```

Comment:
Can we add new items in v1beta4? 

Or should we maintain a list of changes to the configurations somewehere?
https://github.com/kubernetes/kubernetes/blob/491ed8df11bdf86340f7424f03406129e09f3fe6/cmd/kubeadm/app/apis/kubeadm/v1beta4/doc.go#L25-L29

Reply:
also i don't know if the docs team regenerates these on every release
https://kubernetes.io/docs/reference/config-api/kubeadm-config.v1beta4/

i think yes